modules/programs/joplin-desktop.nix

```
@@ -0,0 +1,129 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.programs.joplin-desktop;
+  # config path is the same for linux and mac
+  configPath = (config.home.homeDirectory + "/.config/joplin-desktop");
+in {
+  options.programs.joplin-desktop = {
+    enable = lib.mkEnableOption "joplin-desktop";
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.joplin-desktop;
+      defaultText = lib.literalExpression "pkgs.joplin-desktop";
+      description = "Package containing the joplin-desktop program.";
+    };
```

Can use the `mkPackageOption` function. Also please separate options by empty lines:

```suggestion

    package = lib.mkPackageOption pkgs "joplin-desktop" { };

```

---

Thx, I'll do that now